crates/integration_tests/Scarb.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "integration_tests"
```

Comment:
Why not just `tests`?

Reply:
Actually it probably makes sense to have client as a separate package (binary), and consensus left as a library.